README.md

```diff
@@ -172,3 +199,109 @@ If you develop in Typescript, the type checker will help prevent you from creati
 ```typescript
 const listCV = new ListCV([new TrueCV, new IntCV(1)]);
 ```
+
+## Post Conditions
+Three types of post conditions can be added to transactions:
```

Comment:
Would be helpful to link to https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md#transaction-post-conditions